### PR TITLE
Add RCS resources

### PR DESCRIPTION
**Pull request by cwfoo**
## What does this PR do?
Add [Revision Control System (RCS)](https://en.wikipedia.org/wiki/Revision_Control_System) resources.

### Description

Add two resources:
* *RCS — A System for Version Control* — The paper where the original author of RCS describes RCS.
* *RCS Handbook* — A book that shows how to use RCS.

### Why is this valuable (or not)?

*RCS Handbook* is particularly valuable because it goes far beyond the man pages and the short tutorials found on various websites. For example, the book teaches about branching, merging, and managing software projects.

### How do we know it's really free?

These texts are made available on the [Purdue RCS Homepage](https://www.cs.purdue.edu/homes/trinkle/RCShome/#DOC).

### For book lists, is it a book? For course lists, is it a course? etc.

One paper and one book.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2074,6 +2075,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
 
 
+### RCS
+
+* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PS)
+* [RCS Handbook](https://www.cs.purdue.edu/homes/trinkle/RCShome/contrib/odonavan/RCS_Book.tar.Z) - Brian O’Donovan
```
Comment (davorpa) [inline books/free-programming-books-langs.md:2081]: Should we accept compressed files?

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2074,6 +2075,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
 
 
+### RCS
+
+* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PS)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:2080]: Fixing file format notes

```suggestion
* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PostScript)
```

Are there any other format easier readable like PDF, ePub...?

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by davorpa** [comment]
If is a version control system, remember reorganize via #5526

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (cwfoo) [reply books/free-programming-books-langs.md:2080]: FreeBSD has a PDF version: https://docs.freebsd.org/44doc/psd/13.rcs/paper.pdf

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (LuigiImVector) [reply books/free-programming-books-langs.md:2081]: There are some link that redirect to a compressed files so I think we accept those.

Example:
https://github.com/EbookFoundation/free-programming-books/blob/3d3929bc781106e4f18cd90d8f0867c8163dd8a9/books/free-programming-books-langs.md#L353

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (LuigiImVector) [reply books/free-programming-books-langs.md:2080]: I think that it would be better to replace the .ps link with the GNU link

**Review by eshellman — COMMENTED** [review]
It's not our goal to document 40 year old software "primarily of historical interest". Wikipedia has a good article on it. RCS is not worth a section of its own as far as I can tell. If there were a book on it we would include it in our CVS section, if we even had a CVS section. or our "history of software" section if we had that.

Also RCS is today too obscure an acronym for us to use as a heading.

Thank you for the suggestion; it is appreciated even if not accepted!

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (davorpa) [reply books/free-programming-books-langs.md:2080]: > @davorpa GNU has a PDF version: [gnu.org/software/rcs/tichy-paper.pdf](https://www.gnu.org/software/rcs/tichy-paper.pdf) FreeBSD too: [docs.freebsd.org/44doc/psd/13.rcs/paper.pdf](https://docs.freebsd.org/44doc/psd/13.rcs/paper.pdf)

I agree with Luigi and Eric attending to contrib guidelines, using PDF or ePUB, and even use a gnu.org as target domain. Meanwhile... I'd opt to fix file format notes with the proposed name

**Comment by eshellman** [comment]
What do you think, @cwfoo ? (You were tied for sixth for most F-P-B contributions in 2021!)

**Comment by cwfoo** [comment]
@eshellman
> It's not our goal to document 40 year old software "primarily of historical interest". Wikipedia has a good article on it. RCS is not worth a section of its own as far as I can tell. If there were a book on it we would include it in our CVS section, if we even had a CVS section. or our "history of software" section if we had that.

I occasionally use RCS myself, but I completely understand that RCS is widely considered obsolete these days (it was long ago removed from the base system of the BSDs). In any case, the "books" submitted in this pull request can easily be found through the most obvious webpages:

* https://www.gnu.org/software/rcs/
* https://www.cs.purdue.edu/homes/trinkle/RCS/

> Thank you for the suggestion; it is appreciated even if not accepted!

This pull request can now be closed. This pull request can be revisited in the future when there are more appropriate sections in the book lists.

> (You were tied for sixth for most F-P-B contributions in 2021!)

This is most surprising. :laughing: Only 7 commits!

**Pull request closed without merging**